package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "wire-server",
+  "license": "GPL-3.0",
+  "private": true,
+  "scripts": {
+    "emails": "python services/brig/deb/opt/brig/bin/fetch.py"
```

Comment:
`"emails": "node services/brig/deb/opt/brig/bin/fetch.js"` -> JavaScript all the things!

Reply:
<img width="989" alt="screen shot 2018-02-21 at 11 39 55" src="https://user-images.githubusercontent.com/125676/36472999-fe1d87ba-16fb-11e8-85cb-9cfb4e4cdad1.png">

:)

Reply:
Exactly, JavaScript is missing there^^